USAGE.md

```diff
@@ -0,0 +1,36 @@
+# STARTTLS Policy List Usage Guidelines
+
+## MTA-STS interoperation for sending mailservers
+
+As explained in previous posts, the ideal is for the STARTTLS policy list to act as a "preload list" for MTA-STS domains. Although there are many parallels to the web situation with HSTS / HSTS preload list, this list is not an exact mirror. There are a couple of edge cases to consider when implementing both MTA-STS and the policy list. This describes the expected behavior of **sending** mailservers that both validate MTA-STS and follow the policy list.
```

Comment:
I think "previous posts" isn't really applicable in the context of this README. :slightly_smiling_face: